packages/taquito/src/operations/rollup-origination-operation.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  OperationContentsAndResult,
+  OperationContentsAndResultTxRollupOrigination,
+} from '@taquito/rpc';
+import { Context } from '../context';
+import { Operation } from './operations';
+import {
+  FeeConsumingOperation,
+  ForgedBytes,
+  GasConsumingOperation,
+  RPCTxRollupOriginationOperation,
+  StorageConsumingOperation,
+} from './types';
+
```

Comment:
maybe we could change the filename for this one to have a `tx-` prefix as well to future-proof sc-rollup names potentially?